rust/sbp/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "sbp"
+version = "0.1.0"
+authors = ["Gareth McMullin <gareth@swiftnav.com>"]
```

Comment:
Should we put a more general email address here? `dev@swiftnav.com`? I guess it's somewhat up to @gsmcmullin if he wants to be the point of contact for our first rust crate :)

Reply:
I don't know where `dev@swiftnav.com` goes, but it seems more appropriate.